geth/api/backend.go

```diff
@@ -23,7 +29,7 @@ type StatusBackend struct {
 	accountManager common.AccountManager
 	txQueueManager common.TxQueueManager
 	jailManager    common.JailManager
-	// TODO(oskarth): notifer here
+	notification   common.Notifier
```

Comment:
I think having this field named `notifier` would ease understanding. Because it's not a single notification, it's a notifier which may send many notifications.

Reply:
Ok.